cmd/kops/update_cluster.go

```diff
@@ -282,11 +282,11 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 			fmt.Fprintf(sb, " * validate cluster: kops validate cluster\n")
 			fmt.Fprintf(sb, " * list nodes: kubectl get nodes --show-labels\n")
 			if !usesBastion(instanceGroups) {
-				fmt.Fprintf(sb, " * ssh to the master: ssh -i ~/.ssh/id_rsa admin@%s\n", cluster.Spec.MasterPublicName)
+				fmt.Fprintf(sb, " * ssh to the master: ssh -A -i ~/.ssh/id_rsa admin@%s\n", cluster.Spec.MasterPublicName)
```

Comment:
Is this the only place we print this? Also, we have another issue open about the user vs the os of the nodes.  Up to you if you want to fix it here as well.